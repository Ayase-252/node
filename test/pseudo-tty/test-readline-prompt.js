'use strict';

const common = require('../common');
const assert = require('assert');
const rl = require('readline');

(async () => {
  const answer = await rl.prompt('what is your favorite food?');
  assert.strictEqual(answer, 'sushi');
})().then(common.mustCall());
