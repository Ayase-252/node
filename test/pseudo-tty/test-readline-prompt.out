sushi
[1G[0Jwhat is your favorite food?
 [1Gsushi
